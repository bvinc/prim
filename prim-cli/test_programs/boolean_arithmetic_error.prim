fn main() {
    let value = true + false
    println(value)
}
