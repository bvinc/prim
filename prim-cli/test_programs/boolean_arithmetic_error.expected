ERROR: Type check error: Invalid binary operands for '+' (left: bool, right: bool)
